modules/test/playwright/tests/commerce/commerce-account-web/commerceOrganizationManagement.spec.ts

```diff
@@ -27,6 +27,94 @@ async function waitForAnimationEnd(locator: Locator) {
 	return handle?.dispose();
 }
 
+test('LPD-44149 Can search an entry present multiple times in organization management', async ({
+	apiHelpers,
+	organizationManagementPage,
+	usersAndOrganizationsPage,
+}) => {
+	const organization1 = await apiHelpers.headlessAdminUser.postOrganization({
+		name: `Org${getRandomInt()}`,
+	});
+
+	const organization2 = await apiHelpers.headlessAdminUser.postOrganization({
+		name: `Org${getRandomInt()}`,
+	});
+
+	const organization3 = await apiHelpers.headlessAdminUser.postOrganization({
+		name: `Org${getRandomInt()}`,
+		parentOrganization: {
+			externalReferenceCode: organization1.externalReferenceCode,
+		},
+	});
+
+	const accounts = [];
+	const emailAddresses = [
+		'buyer@liferay.com',
+		'test@liferay.com',
+		'buyer@liferay.com',
+	];
+	const organizationIds = [
+		organization3.id,
+		organization2.id,
+		organization2.id,
+	];
```

Comment:
put his in a single array with objects, like it was before
```
const data = [
  {
    emailAddress: value;
    organizationId: value;
  }, 
  ....
]
```
so we can just loop on a single array extracting what we need from the object